platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -119,11 +108,9 @@
     @Mock
     private Iterator<Path> configFilesIterator;
 
-    @Mock
-    private Path configPath1;
+    private Path configPath1 = Paths.get("/root/etc/failed/pid.config");
```

Comment:
Can we change the `configPath1`'s path to `/root/etc/pid1.config` and the path for `configPath2` be `/root/etc/pid2.config`?